setup.py

```diff
@@ -12,7 +12,7 @@ def read(fname):
 
 setup(
     name="pyotgw",
-    version="0.4b1",
+    version="0.4b2",
```

Comment:
Version will be bumped on release, please don't bump it in a PR.

Reply:
Resolved in ac3c3ab